src/Stratis.Bitcoin.Features.MemoryPool/MempoolFeature.cs

```diff
@@ -128,22 +128,21 @@ public static void BuildDefaultConfigurationFile(StringBuilder builder, Network
         /// <inheritdoc />
         public override void Dispose()
         {
-            if (this.mempoolManager != null)
+            if (this.mempoolManager == null) return;
```

Comment:
And that even happen?

Reply:
I guess it won't even initialise if mempool manager is null. We should probably guard it in the constructor.